LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Creative Commons
+Copyright (c) 2022 WordPress Foundation
```

Comment:
I think actually this should be "the contributors" to match Gutenberg and core's license text:

https://github.com/WordPress/gutenberg/blob/trunk/LICENSE.md

https://github.com/WordPress/wordpress-develop/blob/trunk/src/license.txt#L3